test/dart3_sdk_constraint_hack_test.dart

```diff
@@ -142,7 +142,7 @@ void main() {
     });
     await pubGet(
       error: contains(
-        'The lower bound of ">=2.10.0 <3.0.0" does not enable null safety.',
+        'The lower bound of ">=2.10.0 <3.0.0" must be 2.12.0 or higher for null safety.',
```

Comment:
Should we suggest a 2.19 sdk for migrating?
```suggestion
        '''
 The lower bound of ">=2.10.0 <3.0.0" must be 2.12.0 or higher for null safety.
 
 Consider using a Dart 2.19 SDK to migrate to null safety.''',
```

Reply:
Maybe we're better off adding something like `For details, see https://dart.dev/go/null-safety` ?

Reply:
Yes - that is even better!

Reply:
Done